_core/smart/neon-node.md

```diff
@@ -27,64 +27,57 @@ git clone https://github.com/blockstack/stacks-blockchain.git
 
-### Setup your Proof-of-Burn miner
+First, find the `bootstrap_node` line in the `[node]` section, and update it to the following:
 
-In this phase of the testnet, miners will be burning BTC to participate in leader election. In order to mine, you'll need to configure your node with a BTC keychain, and you'll need to fund it with BTC. We're providing a BTC faucet to make it easy to run a miner.
+```toml
+bootstrap_node = "048dd4f26101715853533dee005f0915375854fd5be73405f679c1917a5d4d16aaaf3c4c0d7a9c132a36b8c5fe1287f07dad8c910174d789eb24bdfb5ae26f5f27@35.245.47.179:20444"
+```
 
-Let's start by generating a keypair:
+Next, find the `peer_host` line in the `[burnchain]` section, and update it to:
 
-```bash
-cargo run --bin blockstack-cli generate-sk --testnet
-
-# Output
-# {
-#  secretKey: "b8d99fd45da58038d630d9855d3ca2466e8e0f89d3894c4724f0efc9ff4b51f001",
-#  publicKey: "02781d2d3a545afdb7f6013a8241b9e400475397516a0d0f76863c6742210539b5",
-#  stacksAddress: "ST2ZRX0K27GW0SP3GJCEMHD95TQGJMKB7G9Y0X1MH"
-# }
+```toml
+peer_host = "35.245.47.179"
 ```
 
-**TODO**: guidance / update on generating a BTC address from this keypair.
+### Run your node
 
-Once you have your BTC address, head over to our testnet website to use the BTC faucet:
+You're all set to run a node that connects to the Neon network.
 
-**TODO**: URL, guidance for BTC faucet
+Back in the command line, run:
 
-Now that you have some testnet BTC, you'll need to configure your Stacks node to use this wallet.
+```bash
+cargo testnet ./testnet/follower-conf.toml
+```
 
-Open up the file `testnet/Stacks.toml`. Find the section that starts with `[burnchain]`. Update that section so that it looks like this:
+You should see some logs that look something like the this:
 
-```toml
-[burnchain]
-chain = "bitcoin"
-mode = "neon"
-peer_host = "127.0.0.1" # todo(ludo): update URL with neon.blockstack.org when deployed
-burnchain_op_tx_fee = 1000
-commit_anchor_block_within = 10000
-rpc_port = 3000
-peer_port = 18444
+```
+Starting testnet with config ./testnet/follower-conf.toml...
+Transactions can be posted on the endpoint:
+POST http://127.0.0.1:9001/v2/transactions
+INFO [1587602447.879] [src/chainstate/stacks/index/marf.rs:732] First-ever block 0f9188f13cb7b2c71f2a335e3a4fc328bf5beb436012afca590b1a11466e2206
 ```
 
-**TODO**: update `peer_host`, and how do you specify your BTC keychain?
+Awesome! Your node is now connected to the Neon network. Your node will receive new blocks when they are produced, and you can use your node's RPC API to send transactions, fetch information for contracts and accounts, and more.
 
-### Running your mining node
+### Creating an optimized binary
 
-Now that you're all set up, you can run your miner. In the command line, run:
+The steps above are great for trying to run a node temporarily. If you want to host a node on a server somewhere, you might want to generate an optimized binary. To do so, use the same configuration as above, but run:
```

Comment:
do we want to guide this part more? host where? how to? any limitations or requirements? etc

Reply:
I would say that's out of scope for getting this released. And we are still figuring that out for ourselves. All of this is very much in-flux, and will change rapidly with mining, mainnet, etc.